agent/src/main/java/com/teamscale/jacoco/agent/Agent.java

```diff
@@ -33,8 +35,8 @@
 	protected final IXmlStore store;
 
 	/** Constructor. */
-	/*package*/ Agent(AgentOptions options) throws IllegalStateException, UploadStoreException {
-		super(options);
+	/*package*/ Agent(AgentOptions options, IAgent agent) throws IllegalStateException, UploadStoreException {
```

Comment:
Reame to `agentDelegate` or `jacocoAgent`? Looks strange if the agent gets an agent as parameter 😉

Reply:
done